CONTRIBUTING.md

```diff
@@ -56,27 +56,46 @@ Merging Pull Requests (for project contributors with write access)
 
 ### Building and Testing
 
-The easiest way to build is by running the following command from the root folder:
+>**Note**: Build and test instructions have recently changed. The ```pom.client.xml``` file is no
+> longer in the root of the repository and should be not be used as an aggregator moving forward.
+
+This repository uses Maven to build the various modules that make up the Azure SDK for Java. You can build the entire set of modules by running the following command from the root of the repository:
+
 ```
-mvn -f pom.client.xml -Dgpg.skip -DskipTests -Dinclude-non-shipping-modules clean install
+mvn -Dgpg.skip clean install
 ```
-- `-f pom.client.xml`: tells maven to target latest Azure SDK for Java project.
-- `-Dgpg.skip`: disables [gpg](https://mran.microsoft.com/snapshot/2016-12-19/web/packages/gpg/vignettes/intro.html) signing.
-- `-DskipTests:` Building without running unit tests would speed operation up, however, make sure all tests pass before creating a new PR.
-- `-Dinclude-non-shipping-modules:` Installing and Runing sdk build tools.
-- `clean:` will remove any previous generated output.
-- `install:`  compiles project and installs it in the local Maven cache.
 
->**Note**: Refer to [wiki](https://github.com/Azure/azure-sdk-for-java/wiki/Building) for learning about how to build using Java 11 
->and [this wiki](https://github.com/Azure/azure-sdk-for-java/wiki/Unit-Testing) for guidelines on unit testing
+This will build, test and install all of the modules into your local Maven cache. Optionally, you
+can also skip unit test execution by issuing the following command. This can be useful when you
+just want to try out a quick change without having to worry about unit tests.
 
-### Compiling one project only
+```
+mvn -Dgpg.skip -DskipTests clean install
+```
 
+If you only want to work on one module, you may opt to build just that module and its dependencies (e.g. com.azure:azure-messaging-eventhubs):
+
+```
+mvn -Dgpg.skip -DskipTests -pl com.azure:azure-messaging-eventhubs -am
```

Comment:
Is it better to have another example on how to build two modules at once?
e.g.
-pl com.azure:azure-storage-blob, com.azure:azure-storage-file

Reply:
Added an example.